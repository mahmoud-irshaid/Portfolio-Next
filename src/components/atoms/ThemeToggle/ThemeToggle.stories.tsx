/* eslint-disable import/no-anonymous-default-export */
import { ThemeToggle } from './ThemeToggle';

export default {
	title: 'atoms/ThemeToggle',
	component: ThemeToggle,
};

export const ThemeToggleStory = () => <ThemeToggle />;

ThemeToggleStory.storyName = 'ThemeToggle';
