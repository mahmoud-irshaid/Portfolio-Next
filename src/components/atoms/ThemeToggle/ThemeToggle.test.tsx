import { ThemeToggle } from '../ThemeToggle';

import { render } from '@testing-library/react';

describe('ThemeToggle', () => {
	it('renders correctly', () => {
		const { container } = render(<ThemeToggle />);
		expect(container).toMatchSnapshot();
	});
});
