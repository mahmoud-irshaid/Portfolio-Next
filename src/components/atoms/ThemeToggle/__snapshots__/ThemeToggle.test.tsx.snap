// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeToggle renders correctly 1`] = `
<div>
  <button
    aria-label="Toggle Dark Mode"
    class="flex items-center justify-center rounded-full border-2 border-grey-300 bg-transparent p-2 ring-grey-300 transition-all hover:ring-2 hover:ring-offset-2 dark:border-grey-700 dark:bg-transparent dark:ring-grey-200 dark:hover:ring-2 dark:hover:ring-offset-2"
    type="button"
  >
    <svg
      class="h-5 w-5 text-grey-800 dark:text-grey-200"
      fill="none"
      stroke="currentColor"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
      />
    </svg>
    <span
      class="md:hidden"
    >
      Switch to 
      dark
       mode
    </span>
  </button>
</div>
`;
